Examples/Tests/load_external_field/inputs_test_3d_load_external_field_particle_multi_time_picmi.py

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/env python3
+from math import pi
+
+from pywarpx import picmi
+
+# -------------------
+# General parameters
+# -------------------
+max_steps = 300
+max_grid_size = 40
+nx = ny = nz = max_grid_size
+
+xmin, xmax = -1.0, 1.0
+ymin, ymax = xmin, xmax
+zmin, zmax = 0.0, 5.0
+
+number_per_cell = 200
+verbose = 1
+dt = 4.4e-7  # seconds
+
+# Two fields with different frequencies: ω and 2ω.
+# B_sum(tN) ∝ cos(pi/3) + cos(2pi/3) = 0.5 - 0.5 = 0
+t_end = max_steps * dt
+omega = (pi / 3.0) / t_end  # cos(ω tN) = +0.5
+phase = 0.0
+path = "../../../../openPMD-example-datasets/example-femm-3d.h5"
+
+# -------------------
+# Species
+# -------------------
+constants = picmi.constants
+ion_dist = picmi.ParticleListDistribution(
+    x=0.0,
+    y=0.2,
+    z=2.5,
+    ux=9.5e-05 * constants.c,
+    uy=0.0 * constants.c,
+    uz=1.34e-4 * constants.c,
+    weight=1.0,
+)
+ions = picmi.Species(
+    particle_type="H",
+    name="proton",
+    charge="q_e",
+    mass="m_p",
+    warpx_do_not_deposit=1,
+    initial_distribution=ion_dist,
+)
+
+# -------------------
+# Applied field (file)
+# -------------------
+applied_field = picmi.LoadAppliedField(
+    load_E=False,
+    load_B=True,
+    B_external_fields={
```

Comment:
Since this deviates from the PICMI standard and only WarpX would understand this syntax, our convention is that the argument should be prefixed with `warpx_`. Could you make the corresponding change throughout the source code, the docstrings, and the tests?
```suggestion
    warpx_B_external_fields={
```